packages/@webex/plugin-meetings/src/locus-info/controlsUtils.ts

```diff
@@ -119,4 +119,22 @@ ControlsUtils.getId = (controls: any) => {
   return null;
 };
 
+/**
+ * Extract the id from the record controls object
+ * @param {LocusControls} oldControls
+ * @param {LocusControls} controls
+ * @returns {String|null}
```

Comment:
issue: The return type is not correct. It returns a boolean

Reply:
Done